config/versions/versions.go

```diff
@@ -211,15 +210,25 @@ func exchangeDeploy(ctx context.Context, patch ExchangeVersion, method func(Exch
 }
 
 // registerVersion takes instances of config versions and adds them to the registry
-func (m *manager) registerVersion(ver int, v any) {
+func (m *manager) registerVersion(ver uint16, v any) {
 	m.m.Lock()
 	defer m.m.Unlock()
-	if ver >= len(m.versions) {
-		m.versions = slices.Grow(m.versions, ver+1)[:ver+1]
+	if int(ver) >= len(m.versions) {
+		m.versions = slices.Grow(m.versions, int(ver+1))[:ver+1]
 	}
 	m.versions[ver] = v
```

Comment:
What about a check for version number collisions too?

Reply:
Used a panic here, because it's significantly more convenient and it'd be a startup issue for a dev anyway.

Fixed https://github.com/gbjk/gocryptotrader/commit/771c689564

Reply:
Nice, that's what I had in mind too 👍